cudapoa/src/cudapoa_batch.cuh

```diff
@@ -462,6 +464,10 @@ protected:
             GW_LOG_WARN("Kernel Error:: Loop count exceeded upper bound in nw algorithm in batch {}\n", bid_);
             output_status.emplace_back(error_type);
             break;
+        case genomeworks::cudapoa::StatusType::exceeded_adaptive_banded_matrix_size:
```

Comment:
I inserted this error message here as code 6 because it made sense for grouping, but it might affect error code tracking. What is the error code strategy? Should it be moved to the end of the list when adding new messages?

Reply:
I thought somewhere @tijyojwad had mentioned new error codes to be added to the bottom of the list. However, I think logical order makes sense, also we have previously removed some redundant error codes, so this won't be the first time the error list is changed.